python/pyspark/sql/types.py

```diff
@@ -196,28 +196,19 @@ class DecimalType(FractionalType):
     """Decimal (decimal.Decimal) data type.
```

Comment:
add documentation for what precision/scale mean, and their default value here. 

We should also update the scaladoc to include those.